deploy/scripts/src/btm/test.ts

```diff
@@ -0,0 +1,60 @@
+import { type Options } from 'k6/options'
+
+import { selectProfile, type ProfileList, describeProfile } from '../common/utils/config/load-profiles'
+
+import { uuidv4 } from 'https://jslib.k6.io/k6-utils/1.4.0/index.js'
+import { AWSConfig, SQSClient } from 'https://jslib.k6.io/aws/0.7.2/sqs.js'
```

Comment:
These will need to be added as files to the repository as we cannot run external code from the performance test stack.

I'd suggest adding the files to a new folder under `/deploy/scripts/common/utils/jslib/`, add any copyright licensing as a comment at the top of the script and then change the imports here to use relative paths.

Reply:
kept the mobile folders structure of k6 to match existing Mobile folder structure